### PR TITLE
feat(styles): new footer design

### PR DESCRIPTION
**Pull request by runrog**

### CODE REVIEW

File: styleguide/derek/global-components/footer/_footer.scss [hunk 1]
```diff
@@ -46,6 +47,7 @@
 .footer-tab-link {
   color: $white;
   font-size: 1.3rem;
+  font-weight: 500;
```
Comment (WidgetsBurritos) [inline styleguide/derek/global-components/footer/_footer.scss]: We only have the 500 weight installed on the rackspace_spa theme, so this change won't make any sort of difference elsewhere on the site.

File: styleguide/derek/global-components/footer/footer.ejs [hunk 2]
```diff
@@ -1,54 +1,72 @@
...
+        </a>
+        <a href="https://twitter.com/Rackspace" target="_blank" class="basement-iconListItemLink">
+          <span class="fa fa-2x fa-twitter"></span>
+        </a>
+        <a href="https://www.linkedin.com/company/rackspace/" target="_blank" class="basement-iconListItemLink">
+          <span class="fa fa-2x fa-linkedin"></span>
+        </a>
+        <a href="https://www.youtube.com/rackspace" target="_blank" class="basement-iconListItemLink">
+          <span class="fa fa-2x fa-youtube"></span>
+        </a>
```
Comment (WidgetsBurritos) [inline styleguide/derek/global-components/footer/footer.ejs]: For accessibility purposes, we'll need text inside the `<span>`.  You can then use CSS to hide that text, but we need it for screen readers. 

So for example: 
```<span class="fa fa-2x fa-facebook-official">Rackspace on Facebook</span>```

File: styleguide/derek/global-components/footer/_footer.scss [hunk 3]
```diff
@@ -21,6 +21,7 @@
 .footer-menu-link {
   color: $gray-light;
   font-size: 1.2rem;
+  font-weight: 500;
```
Comment (WidgetsBurritos) [inline styleguide/derek/global-components/footer/_footer.scss]: We only have the 500 weight installed on the rackspace_spa theme, so this change won't make any sort of difference elsewhere on the site.

**Review by WidgetsBurritos — CHANGES_REQUESTED** [review]

**Comment by runrog** [comment]
[run ci]

File: styleguide/derek/global-components/footer/footer.ejs [hunk 4]
```diff
@@ -1,54 +1,72 @@
...
+    <div class="col-sm-3">
+      <div class="footer-social">
+        <a href="https://www.facebook.com/rackspace" target="_blank">
+          <span class="footer-social-fb">Facebook</span>
+        </a>
+        <a href="https://twitter.com/Rackspace" target="_blank">
+          <span class="footer-social-tw">Twitter</span>
+        </a>
+        <a href="https://www.linkedin.com/company/rackspace/" target="_blank">
+          <span class="footer-social-ln">Linkedin</span>
```
Comment (pobtastic) [inline styleguide/derek/global-components/footer/footer.ejs]: Minor: s/b "LinkedIn" (capital "I").

**Comment by pobtastic** [comment]
LGTM 👍

File: styleguide/derek/global-components/footer/footer.ejs (hunk 4 above)
Comment (runrog) [reply styleguide/derek/global-components/footer/footer.ejs]: I got you!

**Review by WidgetsBurritos — APPROVED** [review]
👍 LGTM

**Pull request merged**